histoqc/BasicModule.py

```diff
@@ -69,3 +69,10 @@ def finalProcessingArea(s, params):
             f"{s['filename']} - After BasicModule.finalProcessingArea NO tissue remains detectable! Downstream modules likely to be incorrect/fail")
         s["warnings"].append(
             f"After BasicModule.finalProcessingArea NO tissue remains detectable! Downstream modules likely to be incorrect/fail")
+
+
+def countTissuePieces(s, params):
+    area_thresh = int(params.get("area_threshold", "1000"))
```

Comment:
I think applying an area threshold here is redundant. Ideally small objects etc. will already be filtered out and the `countTissuePieces` method will just report what's there.